src/components/RequestPanelBasic.vue

```diff
@@ -197,7 +197,7 @@ export default defineComponent({
       const switchCenter =
         postEnabledSwitchTop.value + postEnabledSwitchHeight.value / 2
 
-      return selectCenter != switchCenter
+      return Math.abs(selectCenter - switchCenter) < 0.0001
```

Comment:
Since the original code is `selectCenter != switchCenter`, it should use `>` operator here.

```suggestion
      return Math.abs(selectCenter - switchCenter) > 0.0001
```